### PR TITLE
minio server

### PR DESCRIPTION
**Pull request by huubnijs**
This minio-server installation should preserve current version *and* config (replace:false). But it is a valid installation set of it self.
The use of 'minio-user' for the process is by-passed, and in line with the current implementation it is all as user 'root'.

### CODE REVIEW

File: salt/server/salt.sls [hunk 1]
```diff
@@ -1,9 +1,9 @@
 salt_pkgrepo:
   pkgrepo.managed:
     - humanname: SaltStack
-    - name: deb http://repo.saltstack.com/apt/ubuntu/16.04/amd64/2018.3 xenial main 
+    - name: deb https://repo.saltstack.com/apt/ubuntu/18.04/amd64/2019.2 bionic main
     - file: /etc/apt/sources.list.d/saltstack.list
-    #- key_url: https://repo.saltstack.com/apt/ubuntu/16.04/amd64/2018.3/SALTSTACK-GPG-KEY.pub 
+    #- key_url: https://repo.saltstack.com/apt/ubuntu/18.04/amd64/2019.2/SALTSTACK-GPG-KEY.pub
```
Comment (mjs) [inline salt/server/salt.sls]: why is this commented out?

File: salt/server/salt.sls [hunk 2]
```diff
@@ -1,9 +1,9 @@
 salt_pkgrepo:
   pkgrepo.managed:
     - humanname: SaltStack
-    - name: deb http://repo.saltstack.com/apt/ubuntu/16.04/amd64/2018.3 xenial main 
+    - name: deb https://repo.saltstack.com/apt/ubuntu/18.04/amd64/2019.2 bionic main
```
Comment (mjs) [inline salt/server/salt.sls]: Is jumping to 2019.2 safe?

File: salt/server/salt.sls [hunk 3]
```diff
@@ -1,9 +1,9 @@
 salt_pkgrepo:
```
Comment (mjs) [inline salt/server/salt.sls]: did you mean to include the salt update in this PR?

File: salt/server/tools/map.jinja [hunk 4]
```diff
@@ -7,4 +7,16 @@
...
 grain='id',
-)%}
+) %}
+
+{% set minio = salt['grains.filter_by']({
+  '*-test-*': {
+    'version': 'RELEASE.2017-10-27T18-59-02Z',
+  },
+  '*-prod-*': {
+    'version': 'RELEASE.2017-10-27T18-59-02Z',
```
Comment (mjs) [inline salt/server/tools/map.jinja]: wow this is old now :)

File: salt/server/salt.sls (hunk 1 above)
Comment (huubnijs) [reply salt/server/salt.sls]: I meant it to be 2 pull requests. I mixed them by accident
*  The key url was commented out 10 months ago: "Passing key_url to pkgrepo.installed is unreliable"
* mmm, I don't know if 2019.2 is save, I will return to 2018.3

**Pull request merged**